### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by ervanalb**

### CODE REVIEW

File: resources/qml/application.qml [hunk 1]
```diff
@@ -138,40 +76,159 @@ ApplicationWindow {
...
                         graph: graph;
                         registry: registry
                         context: defaultContext
-                        width: 200
+                        implicitWidth: splitView.openWidth
+                        clip: true
                         
                         Layout.fillHeight: true
+
+                        Rectangle {
```
Comment (zbanks) [inline resources/qml/application.qml]: The library should have a button to open/close it (like settings does), and the search textbox should probably match the styling of the textboxes in the settings pane

File: resources/qml/application.qml [hunk 2]
```diff
@@ -193,17 +250,37 @@ ApplicationWindow {
...
     }
 
-    Action {
-        id: quitAction
-        text: "&Quit"
-        shortcut: "Ctrl+Q"
-        onTriggered: {
-            saveAction.trigger()
-            Qt.quit()
+    function save() {
```
Comment (zbanks) [inline resources/qml/application.qml]: What's the deal with functions vs. actions? (I just don't know. I believe you if functions are better)

File: resources/qml/application.qml (hunk 1 above)
Comment (ervanalb) [reply resources/qml/application.qml]: I changed the search box styling, I'm torn on the library button. The library will never collapse unless you use : to search, and you can always pull it back open with the mouse.

It's also sort of hard to do, the slidey widget I'm using is deprecated so adding a button will probably mean losing the adjustability.

**Comment by ervanalb** [comment]
should be ready to go.

**Pull request merged**